lib/tasks/fetch.rake

```diff
@@ -37,8 +37,8 @@ task expire_old_listings: :environment do
   end
 end
 
-task clear_memory_if_too_high: :environment do
```

Comment:
This task is not called by anything but having `Rails.cache.clear` in the codebase terrifies me so I choose to remove it in this PR.

Reply:
Yup, I think this was removed in Heroku scheduler but not our code.

Fun fact, some of our more knee-jerk solutions to memory problems _originated_ with a bug in the `Memcachier` service itself, which caused a bunch of problems for us. Once that got resolved we still had a bunch of workarounds left over and it wasn't clear which could actually be removed.